server/controllers/request.js

```diff
@@ -0,0 +1,51 @@
+const Request = require("../models/Request");
+const asyncHandler = require("express-async-handler")
+
+// @route POST /requests/create
+// @desc Create new request
+// @access Public
+exports.createRequest = asyncHandler(async (req, res, next) => {
+  const request = await Request.create(req.body);
```

Comment:
It is a good practice to destructure the request body for security purposes (for example, the user can accept the request manually before it even gets to the sitter)

```
const { sitterId, startDate, endDate } = req.body
```

Reply:
Ah I see